Assets/MixedRealityToolkit-SDK/Features/Input/FocusProvider.cs

```diff
@@ -333,24 +333,14 @@ private void CreateUiRaycastCamera()
             cameraObject.transform.parent = transform;
             cameraObject.transform.localPosition = Vector3.zero;
             cameraObject.transform.localRotation = Quaternion.identity;
+
+            // The raycast camera is used to raycast into the UI scene, it doesn't need to render 
+            // anything so is disabled, however its projection matrix will be used to calculate raycast info 
+            // so it is helpful to have a very close nearclip plane
             uiRaycastCamera = cameraObject.AddComponent<Camera>();
             uiRaycastCamera.enabled = false;
-            uiRaycastCamera.clearFlags = CameraClearFlags.Depth;
-            uiRaycastCamera.cullingMask = CameraCache.Main.cullingMask;
-            uiRaycastCamera.orthographic = true;
-            uiRaycastCamera.orthographicSize = 0.5f;
-            uiRaycastCamera.nearClipPlane = 0.1f;
-            uiRaycastCamera.farClipPlane = 1000f;
-            uiRaycastCamera.rect = new Rect(0, 0, 1, 1);
-            uiRaycastCamera.depth = 0;
-            uiRaycastCamera.renderingPath = RenderingPath.UsePlayerSettings;
-            uiRaycastCamera.targetTexture = null;
-            uiRaycastCamera.useOcclusionCulling = false;
-            uiRaycastCamera.allowHDR = false;
-            uiRaycastCamera.allowMSAA = false;
-            uiRaycastCamera.allowDynamicResolution = false;
-            uiRaycastCamera.targetDisplay = 1;
-            uiRaycastCamera.stereoTargetEye = StereoTargetEyeMask.Both;
+            uiRaycastCamera.nearClipPlane = 0.01f;
+            uiRaycastCamera.farClipPlane = 100f;
```

Comment:
Wondering if this should match the global pointer extents

Reply:
Other than ensuring the raycast begins at a close enough distance to the camera that it doesn’t miss nearby UI, I don’t believe it much matters what these values are in terms of ray casting, particularly farclip

Reply:
Will do some more testing tomorrow to find out whether this is the case

Reply:
Not worried about the near clip, just the far clip

Reply:
Did some more testing, the near and far clip actually have no effect on the raycasting that I can tell. I'll just remove the code that sets them to avoid confusing future readers.

Reply:
Oh, I just saw that it was merged already haha, I'll make a very small PR

Reply:
Sounds good